src/world.rs

```diff
@@ -36,6 +36,15 @@ use crate::{
 ///
 /// The components of entities who have the same set of component types are stored in contiguous
 /// runs, allowing for extremely fast, cache-friendly iteration.
+///
+/// There is a max number of unique entity IDs, which means that there is a max number of live
```

Comment:
```suggestion
/// There is a maximum number of unique entity IDs, which means that there is a maximum number of live
```
No need to abbreviate in prose.